docs/source/features/parallelisms.rst

```diff
@@ -112,6 +130,16 @@ NeMo integrates Tensor Parallelism through the implementation from Megatron Core
 
 For detailed API usage and additional configurations, consult the `Megatron Core Developer Guide <https://docs.nvidia.com/Megatron-Core/developer-guide/latest/api-guide/tensor_parallel.html>`_.
 
+FSDP + Tensor Parallelism
+~~~~~~~~~
+
+NeMo supports FSDP along with tensor parallelism. This is done by restricting the model state sharding to the data-parallel domain.
+Using FSDP with tensor parallelism can be helpful when the model doesn't have sufficient parallelism to deploy on a large scale training system with the data-parallel mapping; e.g., running a model with the global batch size of 1024 on 2048 GPUs.
+Also, tensor parallelism enables FSDP feasibility by reducing the model state size and the activation size per GPU, thus lower the FSDP communication overhead and the activation memory overhead.
+
+FSDP + TP works by enabling FSDP (``fsdp=true``) and setting ``tensor_model_parllel_size > 1``.
```

Comment:
FSDP and TP work by enabling FSDP (``fsdp=true``) and setting ``tensor_model_parllel_size > 1``.